### DISCUSSION TITLE
Lesson 1 : Couldn't find the Store Button.

### DISCUSSION THREAD

**Discussion by CryptoHolik** [smartcontractkit/full-blockchain-solidity-course-py#613] [Q&A] [answered]
Hey everyone,
Hope y'all doing great. 

I have just started the course lately and I could say that it's really great.  Thank you so much Patrick for such an informative course. 
However, when I started the first lesson, I wrote the same code for deploying the contract but I couldn't find the "store" button (Also when I write the word store after function, it doesn't look like in the video but just normal (not green)). Is there anything I can do about it please ? 
I tried checking in Q&A forums but I didn't find anything for this issue so I decided to write about it. Thanks for considering my demand.
<img width="820" alt="My Code" src="https://user-images.githubusercontent.com/96414053/147134013-38702a7c-1456-42bd-a80e-f1e5ac614362.png">
<img width="638" alt="Patricks Code" src="https://user-images.githubusercontent.com/96414053/147134015-a65ceea2-902a-41de-bf59-b5a3e2021d83.png">

**Comment by scottjmarler** [comment, marked as answer]
Im just starting the course myself.  Maybe you need a space after public before {  on line 10.  Also are you missing public on line 8?  Just getting familiar with github noticed your issue thought i would give my 2 cents.  Best of luck!

**Reply by CryptoHolik** [reply]
Thanks for your reply man. I actually figured it out. It was just because I didn't compile before I deployed the contract, I didn't pay attention and thought I did. 
And yeah, actually the space between public and the bracket doesn't have any influence on the code, it's the same with or without space. Furthermore, the other public word in line 8 was actually later when he was making the FavoriteNumber fonction public. 

Thanks bro and good learning with Patrick!

**Reply by cromewar** [reply]
You are both awesome @CryptoHolik @scottjmarler, please feel free to make any other questions during the course.

**Reply by eiyen** [reply]
The same mistake. Thanks for your reminder! @CryptoHolik
